definitions/npm/ramda_v0.26.x/flow_v0.76.x-/test_ramda_v0.26.x_list.js

```diff
@@ -302,8 +302,27 @@ const str: string = "hello world";
     { [k: string]: string }
   ] = _.partition(_.contains("s"), { a: "sss", b: "ttt", foo: "bars" });
 
-  const pl: Array<number | string> = _.pluck("a")([{ a: "1" }, { a: 2 }]);
-  const pl1: Array<number> = _.pluck(0)([[1, 2], [3, 4]]);
+  describe('pluck', () => {
+    it('should works on array of objects as maps', () => {
+      const arr: Array<{ [string]: number | string }> = [{ a: "1" }, { a: 2 }];
+      const pl: Array<number | string> = _.pluck("a")(arr);
```

Comment:
What would `_.pluck('b')(arr)` return?

Reply:
Because of `Array<{ [string]: number | string }` it will be typed as `Array<number | string>` . I think this is mainly due to type as Object as Map doesn`t consider undefined keys